OrbitGl/TimeGraph.h

```diff
@@ -157,6 +162,12 @@ class TimeGraph {
   std::shared_ptr<SchedulerTrack> GetOrCreateSchedulerTrack();
   std::shared_ptr<ThreadTrack> GetOrCreateThreadTrack(ThreadID a_TID);
   std::shared_ptr<GpuTrack> GetOrCreateGpuTrack(uint64_t timeline_hash);
+  std::shared_ptr<GraphTrack> GetOrCreateGraphTrack(uint64_t graph_id);
```

Comment:
nodiscard

Reply:
Let's discuss the use of [[nodiscard]] with the team.

Reply:
In this case, since the function is GetOrCreate, nodiscard wouldn't be required anyway.